db/database.go

```diff
@@ -417,14 +423,18 @@ func (db *Database) DocCount() int {
 }
 
 func installViews(bucket base.Bucket) error {
+
+	syncData := "doc._sync"
```

Comment:
can you add a comment that explains what this is?  also, should it be a constant?